test/tstHeapOperations.cpp

```diff
@@ -160,9 +160,27 @@ BOOST_AUTO_TEST_CASE(min_heap)
   BOOST_TEST(a == ref, tt::per_element());
 }
 
+BOOST_AUTO_TEST_CASE(make_heap)
+{
+  for (auto v : {
+           std::vector<int>{},
+           std::vector<int>{6},
+           std::vector<int>{2, 1},
+           std::vector<int>{1, 6, 2, 2, 9, 4, 16},
+           std::vector<int>{8, 6, 7, 2, 0},
+           std::vector<int>{3, 3, 3, 3, 3, 1},
+           std::vector<int>{36, 19, 25, 17, 3, 7, 1, 2, 9},
```

Comment:
```suggestion
           std::vector<int>{36, 19, 25, 17, 3, 7, 1, 2, 9}, // <- is a heap
```

Reply:
Aaaaahhhh!!! Applying suggested change screwed style. Just going to amend one of the commits then, and apply the change.